lib/resque/scheduler/lock/resilient.rb

```diff
@@ -43,7 +43,7 @@ def locked_sha(refresh = false)
           @locked_sha = nil if refresh
 
           @locked_sha ||=
-            Resque.redis.script(:load, <<-EOF.gsub(/^ {14}/, ''))
+            Resque.redis.redis.script(:load, <<-EOF.gsub(/^ {14}/, ''))
```

Comment:
I know.

```
2.3.3 :001 > Resque.redis.class
 => Resque::DataStore 
2.3.3 :002 > Resque.redis.redis.class
 => Redis 
```

Calling `Resque.redis.script` delegates to `Redis::Namespace`, `Resque.redis.redis.script` delegates to `Redis`.  Scripts do not need a namespace.